MediaBrowser.Providers/Plugins/MusicBrainz/MusicBrainzArtistProvider.cs

```diff
@@ -0,0 +1,136 @@
+#pragma warning disable CS1591
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Xml;
+using Diacritics.Extensions;
+using MediaBrowser.Controller.Entities.Audio;
+using MediaBrowser.Controller.Providers;
+using MediaBrowser.Model.Entities;
+using MediaBrowser.Model.Providers;
+using MediaBrowser.Providers.Plugins.MusicBrainz;
+using MetaBrainz.MusicBrainz;
+using MetaBrainz.MusicBrainz.Interfaces.Entities;
+using Microsoft.Extensions.Logging;
+
+namespace MediaBrowser.Providers.Music
+{
+    public class MusicBrainzArtistProvider : IRemoteMetadataProvider<MusicArtist, ArtistInfo>
+    {
+        private Query _musicBrainzQuery;
+
+        public MusicBrainzArtistProvider()
+        {
+            _musicBrainzQuery = new Query();
+
+            Current = this;
+        }
+
+        internal static MusicBrainzArtistProvider Current { get; private set; }
```

Comment:
Where is this used?

Reply:
Not sure, it was there before, so I just assumed it was from the album provider and was there before (I just cleaned up the artist provider to have all the standard fields like name and such).

I assumed it was needed by the plugin engine or something.

If it's not, we can remove it from both providers.

Reply:
It's just a manual way of handling singleton classes. If it has no references you can remove it afaik.